JS-Shapes/script.js

```diff
@@ -0,0 +1,21 @@
+let radius = 5;
+getAreaOfCircle = (radius) => {
+    return 2 * radius * 3.14;
```

Comment:
3.14 is ok
You can also use JavaScript's built-in pi value, `Math.Pi`

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

```suggestion
    return 2 * radius * Math.Pi
```